src/librustc_mir/dataflow/at_location.rs

```diff
@@ -204,10 +204,23 @@ where
     T: HasMoveData<'tcx> + BitDenotation<Idx = MovePathIndex>,
 {
     pub fn has_any_child_of(&self, mpi: T::Idx) -> Option<T::Idx> {
+        // We handle the examination of `mpi` before the loop for two reasons:
+        // - it's a little different from the loop case (due to the
+        //   non-handling of siblings);
+        // - ~99% of the time the loop isn't reached, and this code hot, so we
+        //   don't want to allocate `todo` unnecessarily.
+        // the time
```

Comment:
"the time" here is a dangling fragment.

And since you'll be updating the comment phrasing anyway, i would like it if you added an explicit note that we do not traverse siblings of the input `mpi`, merely its children. (I believe that's what the comment on `push_siblings` was trying to do in the previous version in this code.)